src/main/java/com/thirdcc/webapp/service/DebtService.java

```diff
@@ -1,46 +1,70 @@
 package com.thirdcc.webapp.service;
 
+import com.thirdcc.webapp.domain.enumeration.DebtStatus;
 import com.thirdcc.webapp.service.dto.DebtDTO;
 
 import org.springframework.data.domain.Page;
 import org.springframework.data.domain.Pageable;
 
-import java.util.Optional;
-
 /**
  * Service Interface for managing {@link com.thirdcc.webapp.domain.Debt}.
  */
 public interface DebtService {
 
+//    /**
+//     * Save a debt.
+//     *
+//     * @param debtDTO the entity to save.
+//     * @return the persisted entity.
+//     */
+//    DebtDTO save(DebtDTO debtDTO);
+//    
+//    /**
+//     * Update a debt.
+//     *
+//     * @param debtDTO the entity to update.
+//     * @return the persisted entity.
+//     */
+//    public DebtDTO update(DebtDTO debtDTO);
+//    
+//    /**
+//     * Get all the debts.
+//     *
+//     * @param pageable the pagination information.
+//     * @return the list of entities.
+//     */
```

Comment:
Maybe should remove comments if they are confirmed to be useless in our case

Reply:
Resolved by https://github.com/Five-Fishes/Club-Management-Spring-Server/pull/59/commits/c3c818f0366065911619658628e0fd604ddcfe31